styles/variables.less

```diff
@@ -4,3 +4,5 @@
 // App
 // ---------------------------------------
 @agate-app-keepout: 18px;
+@agate-icon-letter-spacing: normal;
```

Comment:
This file is intended as skin-independent global variables.

These two variables should go into `variables-gallium.less`, and it will propagate to all other skins that haven't also set the variable for themselves.

Reply:
I moved them to the `variables-gallium.less`.

Reply:
Awesome. Thanks, dude!